lib/std/os.zig

```diff
@@ -3745,7 +3746,7 @@ pub fn sendfile(
                     EPIPE => return error.BrokenPipe,
 
                     else => {
-                        _ = unexpectedErrno(err);
+                        const discard = unexpectedErrno(err);
```

Comment:
Why does this need an assignment?

Reply:
because it's an error

Reply:
then should it be: `unexpectedErrno(err) catch {}`?

Reply:
hrm. why don't we support discarding error values (not error *unions*) via assignment to `_`?